pkg/startup/v7/startup.go

```diff
@@ -161,6 +170,7 @@ func (s *startup) writeFiles(role api.AgentPoolProfileRole, w writers.Writer, ho
 			perm = 0644
 		}
 
+		s.log.Debug(fmt.Sprintf("Writing file %s", filepath))
```

Comment:
Is this necessary? Would errors from `MkdirAll` and `WriteFile` would not be enough?